Maple2.Server.Game/Model/Field/Actor/Actor.cs

```diff
@@ -17,16 +20,7 @@ namespace Maple2.Server.Game.Model;
 /// Actor is an entity that can engage in combat.
 /// </summary>
 /// <typeparam name="T">The type contained by this object</typeparam>
-public abstract class Actor<T> : IActor<T>, IDisposable {
-    #region ObjectId
-    private int idCounter;
-
-    /// <summary>
-    /// Generates an ObjectId unique to this specific actor instance.
-    /// </summary>
-    /// <returns>Returns a local ObjectId</returns>
-    protected int NextLocalId() => Interlocked.Increment(ref idCounter);
-    #endregion
+public abstract class Actor<T> : IActor<T>, IActor, IDisposable {
```

Comment:
do you need to add `IActor` here? `IActor<T>` is already here and `IActor<T>` is `IActor`